pkg/kubectl/apply/visitor.go

```diff
@@ -46,6 +46,8 @@ const (
 	SET
 	// DROP drops the field from an object
 	DROP
+	// CONFLICT occurs in recorded and remote
+	CONFLICT
```

Comment:
Why do we want this over just returning ERROR?

Reply:
Instead of doing this, we should have different error subclasses with one for conflict so the user can check the error type.

Reply:
CONFLICT is used as identification by far, since we have not particular error class for conflict. If we are going to add one, I can remove it.